bclientapi/bendoapi.go

```diff
@@ -23,7 +23,11 @@ var (
 func (ia *itemAttributes) GetItemInfo() (*jason.Object, error) {
 	var path = "http://" + ia.bendoServer + "/item/" + ia.item
 
-	r, err := http.Get(path)
+	req, _ := http.NewRequest("GET", path, nil)
+        if ia.token != "" { req.Header.Add("X-Api-Key", ia.token) }
```

Comment:
Oops- well, go fmt will get its chance in a subsequent iteration :)

On Thu, Feb 25, 2016 at 3:46 PM, Don Brower <notifications@github.com>
wrote:

> In bclientapi/bendoapi.go
> <https://github.com/ndlib/bendo/pull/92#discussion_r54161523>:
>
> > @@ -23,7 +23,11 @@ var (
> >  func (ia *itemAttributes) GetItemInfo() (*jason.Object, error) {
> >  	var path = "http://" + ia.bendoServer + "/item/" + ia.item
> >
> > -	r, err := http.Get(path)
> > +	req, _ := http.NewRequest("GET", path, nil)
> > +        if ia.token != "" { req.Header.Add("X-Api-Key", ia.token) }
>
> not sure how i feel having all this on one line. I guess it is okay if
> gofmt left it like this.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ndlib/bendo/pull/92/files#r54161523>.
>